Add Discord to contact informations (#23789)
mdn/content: files/en-us/mdn/community/communication_channels/index.md

---
title: Communication channels
slug: MDN/Community/Communication_channels
page-type: mdn-community-guide
tags:
- meta
- community-guidelines
- governance
---

{{MDNSidebar}}

There are various communication channels using which community can contact MDN Web Docs staff and participate in discussions.

## Chat rooms

> Chat rooms are for queries, clarifications, sharing information, and meetings.

Reviewer: Adding one or two sentences for context 👍🏻

```suggestion
Chat rooms are for queries, clarifications, sharing information, and meetings.
They are also a great place to get to know other community members, share your work, and get help with your contributions.
If you have a question or want to get in touch with us about something, feel free to join us.
```

Author: > Adding one or two sentences for context 👍🏻

```suggestion
Chat rooms are for queries, clarifications, sharing information, and meetings.
They are also great places to get to know other community members, share your work, and get help for your contributions.
If you have a question or want to get in touch with us about something, feel free to join us.
```

### Discord server

> 

Reviewer: Minor hint on the timing as it shows it's fresh

```suggestion
As of January 2023, MDN Web Docs opened up a Discord server to the public.
This server is a great place to see what staff and members of the community are doing on a daily basis and to find out how you can get involved.

```

- [MDN Web Docs community discord server](https://discord.gg/apa6Rn7uEj)

### Matrix chat rooms

> 

Reviewer: ```suggestion
The Matrix chat rooms are an alternative to the Discord server.

```

- [MDN room](https://chat.mozilla.org/#/room/#mdn:mozilla.org) on Mozilla Element web app
- [MDN room](https://app.element.io/#/room/#mdn:mozilla.org) on Matrix Element web app
- [Add-ons room](https://chat.mozilla.org/#/room/#addons:mozilla.org)
- [List of all Mozilla rooms](https://wiki.mozilla.org/Matrix#Commonly_used_rooms)

## GitHub Discussions

> [GitHub Discussions](https://github.com/mdn/mdn-community/discussions) are for conducting polls and for issues that needs a discussion to agree upon a course of action or define an actionable piece of work. Refer [the guidelines](/en-US/docs/MDN/Community/Discussions) before opening a discussion.

Reviewer: ```suggestion
[GitHub Discussions](https://github.com/mdn/mdn-community/discussions) are for conducting polls and for issues that needs a discussion to agree upon a course of action or define an actionable piece of work.
You can use GitHub discussions if you think the answer to your question might be helpful for other people to find in the future.
See [the discussion guidelines](/en-US/docs/MDN/Community/Discussions) before you create a new submission.
```

---
title: Communication channels
slug: MDN/Community/Communication_channels
page-type: mdn-community-guide
tags:
- meta
- community-guidelines
- governance
---

{{MDNSidebar}}

> There are various communication channels using which community can contact MDN Web Docs staff and participate in discussions.

Author: ```suggestion
The community can use one of the many communication channels described here to contact the MDN Web Docs staff, ask questions and participate in discussions.
```

> ## Chat rooms

Author: > Should we order these channels in the order of recommended use? So say, the 'Discord server' first, 'GitHub Discussions' next, and so on? Possibly move Matrix toward the end of the list?

I've grouped them by type(chat, github, forums, email etc.) and sorted based on frequency of use/importance.
![references](https://user-images.githubusercontent.com/87750369/214739985-9f8c3824-3dd4-40f2-97cb-34f36cfdbf68.png)
Looks like our current stance is that discord is in experimentation phase and we are not leaving matrix till everything goes well with discord. So they are on same recommendation level.
Discord has many features so it'll definitely going to be prominent in future.

Chat rooms are for queries, clarifications, sharing information, and meetings.
They are also great places to get to know other community members, share your work, and get help for your contributions.
If you have a question or want to get in touch with us about something, feel free to join us.

### Discord server

As of January 2023, MDN Web Docs opened up a Discord server to the public.
This server is a great place to see what staff and members of the community are doing on a daily basis and to find out how you can get involved.

- [MDN Web Docs community discord server](https://discord.gg/apa6Rn7uEj)

### Matrix chat rooms

The Matrix chat rooms are an alternative to the Discord server.

- [MDN room](https://chat.mozilla.org/#/room/#mdn:mozilla.org) on Mozilla Element web app
- [MDN room](https://app.element.io/#/room/#mdn:mozilla.org) on Matrix Element web app
- [Add-ons room](https://chat.mozilla.org/#/room/#addons:mozilla.org)
- [List of all Mozilla rooms](https://wiki.mozilla.org/Matrix#Commonly_used_rooms)

## GitHub Discussions

> [GitHub Discussions](https://github.com/mdn/mdn-community/discussions) are for conducting polls and for issues that needs a discussion to agree upon a course of action or define an actionable piece of work.
> You can use GitHub Discussions if you think the answer to your question might be helpful for other people to find in the future.
> See [the Discussion guidelines](/en-US/docs/MDN/Community/Discussions) before you create a new submission.

Reviewer 2: I began with a few edits and ended up rewriting a whole lot. Let me know what you think.

```suggestion
[GitHub Discussions](https://github.com/mdn/mdn-community/discussions) on MDN Web Docs is the place to look out for any project-wide [announcements](https://github.com/mdn/mdn-community/discussions/categories/announcements). 

Use GitHub discussions to post your questions and propose your suggestions. Choose the appropriate [discussion category](https://github.com/mdn/mdn-community#github-discussions) when starting a thread. Your answered questions can be really useful for other people looking for similar information in the future.

We strongly recommend you to use GitHub Discussions to share the work you're planning to do and that you think will need a consensus on the course of action. You can also use GitHub Discussions for brainstorming your ideas to define an actionable piece of work, especially when the changes have a wide-ranging impact.

Check out the MDN-specific [discussion guidelines](/en-US/docs/MDN/Community/Discussions) before you create a new submission.
```

## Forums

> The forums can be used for discussing code problems, assessment with the skill tests, assessments that appear on the Learn web development section on MDN, and if you want someone to take a look at your page.

Reviewer 2: ```suggestion
You can use the forums listed below for discussing code problems. You can also request feedback here on your progress with the assessments in the [MDN learning area](https://developer.mozilla.org/en-US/docs/Learn).
```

- [MDN Discourse forum](https://discourse.mozilla.org/c/mdn/236)
- [MDN Discourse forum learning category](https://discourse.mozilla.org/c/mdn/learn/250)
- [Add-ons Discourse](https://discourse.mozilla.org/c/add-ons/35)

## Localization channels

> Each localization team has its own [methods of communication](/en-US/docs/MDN/Community/Contributing/Translated_content).

Reviewer 2: ```suggestion
Each localization team has its own [method of communication](/en-US/docs/MDN/Community/Contributing/Translated_content).
```

> ## Bugs and issues

Reviewer 2: This is not a communication channel per se and should not be listed here. I think for this guidance, folks will need to refer to https://developer.mozilla.org/en-US/docs/MDN/Community/Issues. But this is a good list, maybe we can move it to https://developer.mozilla.org/en-US/docs/MDN/Community/Issues#reporting_an_issue?

Author: done

For reporting bugs and issues related to the MDN Web Docs website use following links:

- [mdn/content](https://github.com/mdn/content/issues)
- [mdn/translated-content](https://github.com/mdn/translated-content/issues)
- [yari](https://github.com/mdn/yari/issues)
- [browser-compat-data](https://github.com/mdn/browser-compat-data/issues)
- [interactive-examples](https://github.com/mdn/interactive-examples/issues)
- [dom-examples](https://github.com/mdn/dom-examples/issues)
- [learning-area](https://github.com/mdn/learning-area/issues)

## Mailing list

> For nonpublic communication email the [mdn-admins list](mailto:mdn-admins@mozilla.org).

Reviewer 2: ```suggestion
For any nonpublic communication, send an email to [mdn-admins](mailto:mdn-admins@mozilla.org).
```
